.tekton/integration-test.yaml

```diff
@@ -0,0 +1,117 @@
+kind: Pipeline
+apiVersion: tekton.dev/v1beta1
+metadata:
+  name: rh-trex-integration-test
+spec:
+  params:
+    - description: 'Snapshot of the application'
+      name: SNAPSHOT
+      default: '{"components": [{"name":"test-app", "containerImage": "quay.io/example/repo:latest"}]}'
+      type: string
+    - description: 'Namespace where the application is running'
+      name: NAMESPACE
+      default: "default"
+      type: string
+    - description: 'Expected output'
+      name: EXPECTED_OUTPUT
+      default: ""
+      type: string
+  tasks:
+    - name: integration-test
+      description: Set up DB, run integration tests, and read results
+      params:
+        - name: SNAPSHOT
+          value: $(params.SNAPSHOT)
+        - name: NAMESPACE
+          value: $(params.NAMESPACE)
+        - name: EXPECTED_OUTPUT
+          value: $(params.EXPECTED_OUTPUT)
+      taskSpec:
+        params:
+        - name: SNAPSHOT
+        - name: NAMESPACE
+        - name: EXPECTED_OUTPUT
+        results:
+        - name: TEST_OUTPUT
+          description: Test output
+        steps:
+        - image: registry.redhat.io/openshift4/ose-cli:latest
+          env:
+          - name: SNAPSHOT
+            value: $(params.SNAPSHOT)
+          - name: NAMESPACE
+            value: $(params.NAMESPACE)
+          - name: EXPECTED_OUTPUT
+            value: $(params.EXPECTED_OUTPUT)
+          script: |
+            # Install Deps
+            dnf -y install jq git make golang podman
+
+            # Runtime Env Config
+            export GOPATH=$HOME/.go
+            export PATH="$PATH:$GOPATH/bin"
+            go install gotest.tools/gotestsum@latest  
+
+            # Set Vars
+            TARGET_COMPONENT_NAME="rh-trex"
+            # REPO_URL=$(echo $SNAPSHOT | jq -r '.components[] | select(.name == "rh-trex").source.git.url')
+            # Temporary workaround - hardcode to make more progress
```

Comment:
just a reminder, we need to remove the temporary workaround once it is ready for merging. Thanks.

Reply:
Removed!